addons/udes_advanced_packaging/models/stock_quant_package.py

```diff
@@ -101,3 +101,43 @@ def write(self, vals):
         """
         self.validate_parent_name(vals)
         return super().write(vals)
+
+    def has_same_content(self, other):
```

Comment:
We can get rid of this, this is used for package swap in v11.

Reply:
fixed in latest commit